elytron/ELY-1712-enhanced-audit-logging.adoc

```diff
@@ -0,0 +1,283 @@
+| Two int values, first being the lower value and second being the higher value, for programmatically, lower value and higher value separated by a comma for markup
+
+| Level
+| FATAL, ERROR, WARN, INFO, DEBUG, TRACE
+| Java String List for programmatically, comma separated list for markup
+
+|===
+
+* Additional audit events should be added for feature parity with Legacy Security Audit Logging. The following tables makes a comparison of Legacy Security Audit Logging features included in, not included in, and if the feature should be added to Elytron Audit Logging:
+|===
+|Legacy Event | Needs to be added to Elytron
+//|Legacy Location
+
+| Rfc5424SyslogEvent
+| Yes
+//| testsuite/shared/.../syslogserver
+
+| CREATED_ASSERTION
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| RESPONSE_TO_SP
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| ERROR_RESPONSE_TO_SP
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| ERROR_SIG_VALIDATION
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| ERROR_TRUSTED_DOMAIN
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| EXPIRED_ASSERTION
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| LOGIN_INIT
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| LOGIN_COMPLETE
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| REQUEST_FROM_IDP
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| REQUEST_TO_IDP
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| RESPONSE_FROM_IDP
+| Yes
+//| picketlink/.../logging, picketlink/.../metrics, picketlink-federation/picketlink/.../audit
+
+| Invalid Authentication
+| Added Already
+//| picketbox/.../AuditUnitTestCase
+
+| Valid Authentication
+| Added Already
+//| picketbox/.../AuditUnitTestCase
+
+| Invalid Permission
+| Added Already
+//|
+
+| Valid Permission
+| Added Already
+//|
+
+| Audit Log Enabled
+| Yes
+//| wildfly-core/.../AuditLogToSyslogTestCase
+
+| Audit Log Disabled
+| Yes
+//| wildfly-core/.../AuditLogToSyslogTestCase
+
+|===
+* Additional customization attributes should be added based on community feedback, listed in the following table:
+|===
+|Feature |Available in Legacy Auditing |Support Case if Applicable |Implementation Required
+
+| "hostname" attribute
+| No
+| https://access.redhat.com/support/cases/#/case/02188481[Case 02188481]
+| Currently supported in Elytron Audit Logging resulting in just needing the attribute to be added to the management configuration
```

Comment:
I already see `host-name` attribute in syslog configuration in Elytron model. How this will be different?

Reply:
Originally I included this because of the support case, but, looking back at it, I think this is out of scope for this RFE and so I'll remove this section.